frontend/common/src/components/UserProfile/ProfileSections/GovernmentInformationSection.tsx

```diff
@@ -55,16 +51,6 @@ const GovernmentInformationSection: React.FunctionComponent<{
                 {intl.formatMessage(getGovEmployeeType(govEmployeeTypeId))}
               </li>
             )}
-            {applicant.interestedInLaterOrSecondment && (
-              <li>
-                {intl.formatMessage({
```

Comment:
Should the corresponding string in fr.json be removed as well for the `intl.formatMessage` removed?

Reply:
removed with [remove additional secondment fr strings](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3698/commits/634cdf2ed537b2ae7b6bca03521808aa3bb97285)